### DISCUSSION TITLE
Would any of you recommend Fulguris as the ideal "Android TV" FOSS Browser?

### DISCUSSION THREAD

**Discussion by SecularSteve** [Slion/Fulguris#453] [Q&A] [answered]
I'm on a quest to find a browser for my Google TV solution that I can trust, and one that works above all else. That includes Remote Control Optimisation.

I just came across this one, and it seems like the ideal Puffin alternative.

**Comment by Slion** [comment]
I never tried it on Android TV so I doubt it would be that good. Though it has been optimized to work well with keyboards so maybe that will help with remote control support. Let us know how it went if you get around trying it out.

**Reply by SecularSteve** [reply, marked as answer]
I just tried it out on an inexpensive Android TV Box solution, and the results are amazing.

![DSC_2134.JPG](https://user-images.githubusercontent.com/33793273/233793690-48b13452-7056-495c-8e9e-32d41a271c2a.JPG)

My ideal post-configured set-up is this:

-General > Desktop Mode On
-Look and Feel > Configuration - Landscape > Set Vertical tab bar Off, Pull to Refresh Off and Desktop width to 100%.

Even if it probably wasn't meant for TVs, it feels like it is without even trying. What amazes me is not only how fast it is, or how little I've had to set up for it, but that those are the only things I've had to configure to make it a bit more ideal for my TV and Remote. Never before has a browser on a TV been this usable and ready from the beginning.

-----

Speaking of which, these are the current problems:

-The lack of a TV Banner/Icon.

I had to sideload it in Aptoide because it won't show up in my Applications probably due to the lack of a Banner Icon and/or Verification.
I'm offering to make it myself if you really want it.

-The lack of a dedicated "Pointer Mode" button in the browser itself.

While the app itself is already so amazingly optimized for Keyboard/Remote use, some sites just won't work without a mouse pointer, even in Mobile modes (especially when Videos are involved).
So I've been thinking of an option to have a button in the taskbar that allows you to switch between "scroll-selecting" and a mouse pointer to move with your arrows for a handful of sites. Kind of like how Desktop mode can be configured with specific web pages.
But that's just my suggestion.

-Pressing the "OK" middle remote button on the web-page bar doesn't do anything.

This last one is probably more of a nitpick, but the common sense idea is that pressing OK on your remote when the web-page bar is highlighted is that it will bring up the keyboard, and you can enter the page manually.
Here however, it does nothing. The only way is to get up into the tabs, and press left to start typing it in. Even then the keyboard won't pop up half the time.
Even if it's probably more or less a minor bug, it should still be looked into.

-A Text Scale Option

For example, the text should be scaled up to 125%, while the rest of the site stays the same size. But that last one is very minor.

-----

But these are all just my opinions/suggestions. I've talked about an ideal Open Source TV Browser solution before, and this is the best one I've experienced so far.

The only other TV Optimised non-Chrome/Opera Universal Browsers you can find these days are Puffin TV (which has a "Monthly Usage Criteria") a few Mainland Chinese fake Open-Source spyware browsers with fake gimmicks (which unfortunately do work) and that built-in browser in the APK Downloader.

These highlighted problems are the only things that need to be resolved, and then you'll have probably the first Certified Open Source Browser ideal for Android/Google TV Boxes.

I'd give it a 9.6/10 (TV Community Certified).
